formats/properties/commonMain/src/kotlinx/serialization/properties/Properties.kt

```diff
@@ -11,6 +11,7 @@ import kotlinx.serialization.descriptors.*
 import kotlinx.serialization.encoding.*
 import kotlinx.serialization.internal.*
 import kotlinx.serialization.modules.*
+import kotlinx.serialization.properties.Properties.Default.toString
```

Comment:
```suggestion

```

Looks like it got here accidentally by referencing toString method in documentation.